src/AppBundle/Service/RestHandler/Report/DeputyCostsEstimateReportUpdateHandler.php

```diff
@@ -29,11 +29,30 @@ public function handle(Report $report, array $data)
         $this
             ->updateHowCharged($report, $data)
             ->updateBreakdownEstimates($report, $data)
-            ->updateMoreInfo($report, $data);
+            ->updateMoreInfo($report, $data)
+            ->updateManagementCost($report, $data)
+        ;
 
         $report->updateSectionsStatusCache([Report::SECTION_PROF_DEPUTY_COSTS_ESTIMATE]);
     }
 
+    /**
+     * @param Report $report
+     * @param array $data
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    private function updateManagementCost(Report $report, array $data) {
```

Comment:
I find it a good practice to have code methods declared in the order that they read - so I think this method should go after `updateMoreInfo` **or** if that calls any methods, then it goes after those methods.

Reply:
That makes sense - I think I'd unconsciously been doing this for a while but not realising. I'll move this down below `updateMoreInfo`.